packages/stable/src/api/geospatial/computeAPI.ts

```diff
@@ -0,0 +1,21 @@
+import { BaseResourceAPI } from '@cognite/sdk-core';
+import {
+  GeospatialComputeRequest,
+  GeospatialComputeResponse
+} from './types';
+
+export class ComputeAPI extends BaseResourceAPI<GeospatialComputeResponse> {
+  /**
+   * [Compute custom json output structures or well known binary format responses based on calculation or selection of feature properties or direct values given in the request.](https://docs.cognite.com/api/v1/#operation/compute)
+   *
+   */
+  public compute = async (
```

Comment:
suggestion(blocking): could you please check the implementation of `search` or `retrieve` features and see if we can use an existing `endpoint` method for not having to use list in our types and having the response already adapter to our need with `items` already extracted.
This is pretty standard in the sdk.

Reply:
I looked through the various methods in the `baseResourceApi.ts` file and couldn't find an existing endpoint method that supported:

* No suffix to url
* No requirement of the query being a filter type
* Doesn't require the response to be a cursor list

Might be missing something, but seems to me like the custom response interface might be needed here (?)